platform/viewer/src/connectedComponents/Viewer.js

```diff
@@ -206,7 +206,119 @@ class Viewer extends Component {
       const patientId = studies[0] && studies[0].patientId;
       const currentTimepointId = this.currentTimepointId;
 
+      // Translate presentation state information into measurement/tool state
+      studies.forEach((study) => {
+      study.seriesList.forEach((series) => {
```

Comment:
We needs to figure out where this logic should go. Right now we have some of it in the Cornerstone extension (for DICOM Seg), and some in the measurement extension (in an ongoing PR for DICOM SR).